src/modules/commentsPopup.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable no-underscore-dangle */
+// import displayComments from './displayComments.js';
```

Comment:
- [ ] Kindly remove unwanted codes completely instead of adding them as comments. It makes your code look kind of messy and prone to bugs :wink: